demos/apps/echo/echo-app-proxy.yaml

```diff
@@ -32,14 +32,14 @@ spec:
         pod.beta.kubernetes.io/init-containers: >
           [{
             "name": "iptables",
-            "image": "gcr.io/istio-testing/init:demo",
+            "image": "docker.io/istio/init:ceade42",
```

Comment:
For the sake of sanity and consistency, can we have one single uber script in the root folder whose job is to figure out the latest stable SHA from dockerhub and stick that into every image? Better still, we could automate this update with @limawhite PR so that we don't end up having different versions of images. Something like istio:stable branch